SinglyLinkedList/SinglyLinkedList.py

```diff
@@ -0,0 +1,136 @@
+"""
+@author: anishukla
+"""
+
+"""Creating a linked list.
+    Adding elements to it.
+    Deleting element from it.
+    Viewing it. """
+
+class Node:    
+    def __init__(self, data): 
+        self.data = data  
+        self.next = None 
+
+class LinkedList: 
+    def __init__(self):  
+        self.head = None
+        
+    def append_start(self, value):
+        newNode = Node(value)
+        newNode.next = self.head
+        self.head = newNode
+        
+    def append_in_between(self, value, position):
+        newNode = Node(value)
+        temp = self.head
+        for i in range(position-2):
+            temp = temp.next
+        newNode.next = temp.next
+        temp.next = newNode        
+    
+    def append_end(self, value):
+        newNode = Node(value)
+        if(self.head == None):
+            self.head = newNode
+        else:
+            temp =  self.head
+            while(temp.next != None):
+                temp = temp.next
+            temp.next = newNode
+    
+    def delete_start(self):
+        if self.head == None:
+            print("Linkedlist is empty")        
+        else:
+            self.head = self.head.next
+    
+    def delete_in_between(self, position):
+        temp = self.head
+        for i in range(position-2):
+            temp = temp.next
+        temp.next = temp.next.next
+        
+    def delete_end(self):
+        temp = self.head
+        while temp.next.next != None:
+            temp = temp.next
+        temp.next = None
+        
+    def view(self):
+        if self.head == None:
+            print("List is empty")
+        else:
+            temp = self.head
+            while temp != None:
+                print(temp.data, end=' ')
+                temp = temp.next
+                
+    def search(self, value):
+        temp = self.head
+        position = 1
+        if temp.data == value:
+            print(position)
+            
+        while temp.next != None:
+            temp = temp.next
+            position = position+1
+            c = 0
+            if temp.data == value:
+                c = c+1
+                print(position)
+                
+        if c == 0:        
+            print("The value isn't present.")
+
+mylist = LinkedList()
+print("Choose 1.To append at start, 2.To append in between, 3.To append at end." )
+print("4.To delete from start, 5.To delete from between, 6. To delete from end.")
+print("7.To View Linkedlist, 8. Search a value's position, 9.To view and exit.")
+
+while(1):
+    K = int(input())
```

Comment:
Hi, thanks for the contribution @Anishukla. Can you do a small change? When we are taking `K` input can print there
```
K = int(input('choose an option between 1 to 9')) 
```